R/models.R

```diff
@@ -114,28 +118,38 @@ predict_model.mgcv <- function(x, newdata) {
   predict(x, newdata)
 }
 
-#' TODO: Add title & description
+#' Define type of model
+#'
+#' @description The function checks whether the model given by \code{x} is
+#' supported, and if it is a regression- or a classification model. If \code{x} is
+#' not a supported model the function will return an error message, otherwise it will
+#' return either \code{"regression"} or \code{"classification"}.
+#'
+#' @inheritParams predict_model
+#'
+#' @details See \code{\link{predict_model}} for more information about
+#' what type of models \code{shapr} currently support.
```

Comment:
Add: 
and the package vignette: \code{vignette("understanding_shapr", package = "shapr")} for how to use a custom model.

Reply:
We don't say anything about using a custom model in the vignette?

Reply:
You are right. I copied this from the documentation in predict_model. I think we should have it there though, along with the combination approach, under a section called "Advanced usage" or something. Agree? If so, I can open a PR and do it. There is a link there that needs to be fixed anyway.

Reply:
Sounds good!